source/index.html.md.erb

```diff
@@ -556,11 +576,11 @@ redirect_uri<br/>(optional) | Redirect URL you [registered](#register-your-app)
 scope<br/>(required) | Space-delimited list of [access scopes](#access-scopes) to which you are asking the user to grant access (note the spaces must be URL-encoded as "%20").
 state<br/>(optional) | Arbitrary string returned to your app; intended to be used by you to ensure that this redirect is indeed from an OAuth flow that you initiated.
 
-Here is an example of a URL to which you would send your user as a `GET` request:
+Here is an example of a URL to send your user as a `GET` request:
  
 `https://app.smartsheet.com/b/authorize?response_type=code&client_id=dheu3dmkd32fhxme&scope=READ_SHEETS%20WRITE_SHEETS&state=MY_STATE`
  
-If your user has not logged in to Smartsheet, he/she will first be directed to the Smartsheet login page. 
+If your user has not logged in to Smartsheet, he/she is first directed to the Smartsheet login page. 
```

Comment:
I generally prefer "they are" to "he/she is"

Reply:
Done.